plugins/home/README.md

```diff
@@ -0,0 +1,106 @@
+# Home
+
+Welcome to the home plugin!
+
+_This plugin was created through the Backstage CLI_
+
+## Overview
+
+The Home plugin introduces a system for composing a Home Page for Backstage in order to surface relevant info and provide convenient shortcuts for common tasks. It's designed with composability in mind with an open ecosystem that allows anyone to contribute with any component, to be included in any Home Page.
+
+For App Integrators, the system is designed to be composable to give total freedom in designing a Home Page that suits the needs of the organization. From the perspective of a Component Developer who wishes to contribute with building blocks to be included in Home Pages, there's a convenient interface for bundling the different parts and exporting them with both error boundary and lazy loading handled under the surface.
+
+## Getting started
+
+If you have a standalone app (you didn't clone this repo), then do
+
+```bash
+# From the Backstage repository root
+cd packages/app
+yarn add @backstage/plugin-home
+```
+
+### Setting up the Home Page
+
+1. Create a Home Page Component that will be used for composition.
+
+`packages/app/src/components/home/HomePage.tsx`
+
+```tsx
+import React from 'react';
+
+export const HomePage = () => {
+  return {
+    /* TODO: Compose a Home Page here */
+  };
+};
+```
+
+2. Add a route where the homepage will live, presumably `/`.
+
+`packages/app/src/App.tsx`
+
+```tsx
+import { HomepageCompositionRoot } from '@backstage/plugin-home';
+import { HomePage } from './components/home/HomePage';
+
+// ...
+<Route path="/" element={<HomepageCompositionRoot />}>
+  <HomePage />
+</Route>;
+// ...
+```
+
+### Creating Components
+
+The Home Page can be composed with regular React components, so there's no magic in creating components to be used for composition 🪄 🎩 . However, in order to assure that your component fits into a diverse set of Home Pages, there's an extension creator for this purpose, that creates a Card-based layout, for consistency between components. The extension creator requires two fields: `title` and `content`. Additionally, you can optionally provide `settings`, `actions` and `contextProvider` as well. These parts will be combined to create a card, where the `content`, `actions` and `settings` will be wrapped within the `contextProvider` in order to be able to access to context and effectively communicate with one another.
```

Comment:
> there's an extension creator for this purpose,

I wonder if its a good idea to reference to somewhere else in the backstage docs to what a "extension creator" is here?